compress/compress.go

```diff
@@ -88,7 +86,6 @@ func NewCompressor(ctx context.Context, logPrefix, outputFile, tmpDir string, mi
 }
 
 func (c *Compressor) Close() {
-	c.datFile.Close()
```

Comment:
here must be close anyway, because compressor.close may be called from defer  - on error or panic ( in cases when  compressor.Compress func will not finish )